### DISCUSSION TITLE
Не получается установить пакет на Nuxt 3.

### DISCUSSION THREAD

**Discussion by bobrov-site** [yandex-maps-unofficial/vue-yandex-maps#216] [Q&A (2.x)] [answered]
Здравствуйте! Возникают трудности при установки Яндекс-карт вашего пакета на Nuxt 3. В документации, указано следующее:

<img width="709" alt="Снимок экрана 2024-02-08 в 14 56 24" src="https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/39225852/abd910e2-f6ad-4e1f-86ee-395e36758f60">

Мне уже сначала, показалось странным добавлять напрямую в modules vue-yandex-maps/nuxt, так как такого пакета нет))

Возможно имелось введу создания файла в modules, но тогда не понимаю как инициализировать. Есть есть еще вариант, через plugins, но не уверен, что это будет корректно в рамках вашей документации.

Буду рад, если поможете разобраться.

**Comment by daniluk4000** [comment]
Здравствуйте! 

Вам необходимо установить vue-yandex-maps@next, после чего прописать так, как у вас на скриншоте. `/nuxt` в данном случае это не пакет, а путь импорта из `vue-yandex-maps@next`

**Reply by daniluk4000** [reply]
Типа

![image](https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/10450717/f35d82bb-ace8-4249-ae97-385046a33ce2)

**Reply by bobrov-site** [reply]
Получаю ошибку vue-yandex-maps.js?v=25d38209:546 An error occured while initializing Yandex Map with onComponentMount setting

Покажу сопровождающий код ради примера - https://github.com/bobrov-site/nedvijka/blob/3306d7dabab87e23202caba513f71a91b934443e/nuxt.config.ts#L7

Здесь инициализация: https://github.com/bobrov-site/nedvijka/blob/f6e5a37ab3fa8c847d774f22a55afdf9872aff6d/pages/search.vue#L16

**Reply by daniluk4000** [reply, marked as answer]
@bobrov-site покажите настройки вашего apikey. Вы точно там прописали в Referrer все нужные домены, включая localhost (или на каком вы поднимаете)? 

Если забыли, то после прописания подождите минут 20, чтобы настройки применились. Если не забыли, посмотрите на запрос в Яндекс - скорее всего он падает на вкладке Сеть

**Reply by bobrov-site** [reply]
Спасибо, помогло! Указал Referrer в качетве локалхоста + добавил YandexMapDefaultSchemeLayer
